system/automation/roles_handler.yaml

```diff
@@ -76,6 +86,50 @@ functions:
     results:
       role: *rvRole
 
+  lookupMembers:
```

Comment:
you either search for members (plural) or lookup for a member (singular)
so this is probably searchMembers, correct?

Reply:
True